SocialCareCaseViewerApi.Tests/V1/IntegrationTests/MASH/UnAssignWorkerFromMashReferralTests.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace SocialCareCaseViewerApi.Tests.V1.IntegrationTests.MASH
+{
+    [TestFixture]
+    public class UnAssignWorkerFromMashReferralTests : IntegrationTestSetup<Startup>
+    {
+        private SocialCareCaseViewerApi.V1.Infrastructure.MashReferral _existingDbReferral;
+
+        [SetUp]
+        public void Setup()
+        {
+            // Clear test database of any rows in the database
+            DatabaseContext.Teams.RemoveRange(DatabaseContext.Teams);
+            DatabaseContext.Workers.RemoveRange(DatabaseContext.Workers);
+            DatabaseContext.MashReferrals.RemoveRange(DatabaseContext.MashReferrals);
+
+            // Create existing referral and unrelated worker
+            var existingDbReferral = IntegrationTestHelpers.SaveMashReferralToDatabase(DatabaseContext, "CONTACT");
+            _existingDbReferral = existingDbReferral;
+        }
+
+        [Test]
+        public async Task SuccessfulPatchUnAssignsWorkerToReferral()
```

Comment:
Since this tests a patch, perhaps we could have an assertion that describes what expect before the patch and then what we expect afterward the patch? 

It's a bit hard to follow from the test alone that there's a guaranteed change in the test. For example, if someone changes the test helper code to not add a worker to the referral, then this test wouldn't fail because it's only testing that the worker is null and has not checked that a worker was definitely assigned to the referral.

Reply:
Added an assertion to check that WorkerId is not null before patch.